docs/user/resources/01-telemetry.md

```diff
@@ -159,27 +159,29 @@ The state of the log components is determined by the status condition of type `L
 | False            | AgentBufferFillingUp         | Buffer nearing capacity. Incoming log rate exceeds export rate. See troubleshooting: [Agent Buffer Filling Up](https://kyma-project.io/#/telemetry-manager/user/02-logs?id=agent-buffer-filling-up)                                |
 | False            | AgentNoLogsDelivered         | Backend is not reachable or rejecting logs. Logs are buffered and not yet dropped. See troubleshooting: [No Logs Arrive at the Backend](https://kyma-project.io/#/telemetry-manager/user/02-logs?id=no-logs-arrive-at-the-backend) |
 | False            | AgentSomeDataDropped         | Backend is reachable, but rejecting logs. Some logs are dropped. See troubleshooting: [No All Logs Arrive at the Backend](https://kyma-project.io/#/telemetry-manager/user/02-logs?id=not-all-logs-arrive-at-the-backend)          |
+| False            | OTTLSpecInvalid              | Invalid <FilterSpec/TransformSpec>: `reason`                                                                                                                                                                                       |
```

Comment:
Compared to the description of other fields, this one is significantly harder to understand:

<img width="1003" height="167" alt="image" src="https://github.com/user-attachments/assets/cfb2aeac-dad9-44ee-b58c-c874e6be9f58" />

Can we try something like this?

```suggestion
| False            | OTTLSpecInvalid              | Invalid OTTL expression in a LogPipeline. Fix the syntax error indicated by the message or see troubleshooting: [Generic EOF Error Message When Using OTTL Transform or Filter](https://kyma-project.io/#/telemetry-manager/user/02-logs?id=generic_eof_error_message_when_using_ottl_transform_or_filter).   |
```